File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 1/src/Ex17.java
//Importando Scanner
import java.util.Scanner;

public class Ex17{
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner ent = new Scanner(System.in);
	
	//vars
	double raio, comprimento, area, volume;
	
	System.out.println("Comprimento, área e volume de uma esfera\n\n");
	
	//Entrada
	System.out.print("Insira o raio da esfera: ");
	raio = ent.nextDouble();
	
	//Processamento
	comprimento = 2 * 3.14 * raio;
	area = 3.14 * Math.pow( raio , 2);
	volume = (3 * 3.14 * Math.pow( raio, 3) ) / 4;
	
	//Saída
	System.out.println("Comprimento = " + comprimento);
	System.out.println("Área = " + area);
	System.out.println("Volume = " + volume );
	
	
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 1/ex17.c
#include <stdio.h>
#include <math.h>

main()
{
    //vars
    float raio, comprimento, area, volume;

    printf("\t\t\tComprimento, area e volume de uma esfera\n\n");

    //Entrada
    printf("Insira o raio da esfera: ");
    scanf("%f",&raio);

    //Processamento
    comprimento = 2 * 3.14 * raio;
    area = 3.14 * pow( raio, 2 );
    volume = 4 * 3.14 * pow( raio, 3 ) / 3 ;

    //Saída
    printf("Comprimento = %.2f\n", comprimento);
    printf("Area = %.2f\n", area);
    printf("Volume = %.2f\n", volume);


}
